### PR TITLE
Formatting and null object error

### PR DESCRIPTION
**Pull request by jeremeguenther**
A couple of formatting change.
Also, in the new custom exception is a null object bug that has been fixed.

### CODE REVIEW

**Comment by jeremeguenther** [comment]
Any chance we can get this pull request merged?

File: Source/DataAccessLayer/Utility.cst [hunk 1]
```diff
@@ -553,7 +553,7 @@ namespace <%=DALNameSpace%>
             CommandText = command.CommandText;
             CommandTimeout = command.CommandTimeout;
             CommandType = command.CommandType.ToString();
-            ConnectionString = command.Connection.ConnectionString;
+            ConnectionString = command.Connection == null ? string.Empty : command.Connection.ConnectionString;
```
Comment (niemyjski) [inline Source/DataAccessLayer/Utility.cst]: Can we change this to `command.Connection != null ? command.Connection.ConnectionString : null;`

File: Source/Entities/EntityInstanceBase.generated.cst [hunk 2]
```diff
@@ -732,10 +733,13 @@ WriteRelationshipPropertyString();
 		/// </returns>
 		public bool IsPropertyChanged(<%= GetClassName(SourceTable, ClassNameFormat.Column) %> column)
 		{
+			if (_originalData == null)
```
Comment (niemyjski) [inline Source/Entities/EntityInstanceBase.generated.cst]: Any reason for this change?

**Comment by niemyjski** [comment]
Yes, sorry for the delay. Can you please review my comments above and I'll get this merged asap.

File: Source/Entities/EntityInstanceBase.generated.cst (hunk 2 above)
Comment (jeremeguenther) [reply Source/Entities/EntityInstanceBase.generated.cst]: If property tracking has not been initialized, then _originalData is null when this method is called and blows up.
I suppose another option would be to throw a custom error telling the user to initialize tracking.

File: Source/DataAccessLayer/Utility.cst (hunk 1 above)
Comment (jeremeguenther) [reply Source/DataAccessLayer/Utility.cst]: I will make this change.  I have a number of additional changes to add to this commit as well.

**Comment by jeremeguenther** [comment]
I stacked another commit on top of this one.  It includes the changes you asked for for the first commit as well as these items.

- Bring some files formatting in line with the Microsoft Visual studio standard.
- fix some names
- Add single column table/view handling to the Utility file
- Add viewcolumns meta data to the EntityViewBase file
- Modify IsPropertyChanged to throw descriptive exception
- Modify NetTiers.cst to handle relative core directory paths.

File: Source/Entities/Views/EntityViewBase.generated.cst [hunk 3]
```diff
@@ -240,18 +241,36 @@ namespace <%=NameSpace%>
...
 		}
-
-		
-		#endregion
 		
+		/// <summary>
+		///	The name of the underlying database view's columns.
+		/// </summary>
+		[BrowsableAttribute(false), XmlIgnoreAttribute()]
+		public string[] ViewColumns
```
Comment (niemyjski) [inline Source/Entities/Views/EntityViewBase.generated.cst]: I'm really not sure why we'd want to ever expose this.

**Comment by niemyjski** [comment]
Changes look good, I'm just not sure why'd we want to expose the view columns.

File: Source/Entities/Views/EntityViewBase.generated.cst (hunk 3 above)
Comment (jeremeguenther) [reply Source/Entities/Views/EntityViewBase.generated.cst]: It currently exposed for tables, just not views, this just brings views in line with how tables are generated.
Also, I use it in reflection when I need to know the original name of the database column before it goes through NetTiers normalization process.  I think I would prefer some sort of dictionary mapping, and maybe some day i'll do that, but for now copying the way tables are handled was enough.

**Comment by niemyjski** [comment]
Thanks for the PR

**Pull request merged**